src/datahike/schema.cljc

```diff
@@ -37,7 +37,11 @@
     :db.type/symbol
     :db.type/uuid
     :db.type/value
-    :db.type/tuple})
+    :db.type/tuple
+    :db.type/cardinality
+    :db.type.install/attribute
+    :db.type/valueType
+    :db.type/unique})
```

Comment:
Why did you add these?

Reply:
These are the allowed types for system attributes. `:db/cardinality` for instance has the allowed values `:db.cardinality/many` and `:db.cardinality/one` as type `.db.type/cardinality`.

Reply:
I see...